### DISCUSSION TITLE
Support for Animated (GIF) Wallpapers

### DISCUSSION THREAD

**Discussion by dcshah2002** [end-4/dots-hyprland#669] [Ideas]
Currently setting GIF wallpaper breaks configuration, it would be great if color generation code could handle GIFs. One way this could be implemented is by using only the first frame in the GIF and using it to generate colors.

**Comment by NextStep-IM** [comment]
If it can be done, I second this.

While we are on this topic, @dcshah2002 where do you get gif wallpapers? I found a website (forgot which one) but it didn't have a good collection.

**Reply by dcshah2002** [reply]
It's hard to find good ones, but I'd suggest to browse on reddit (r/wallpaper)

wallpapercave does have some pixel art gifs I like u can check that out too

**Reply by NextStep-IM** [reply]
Thanks!

**Comment by end-4** [comment]
#727 merged so i'm closing this
